tests/run_tests.py

```
@@ -19,66 +19,71 @@
         except ImportError:
             missing_packages.append(package)
             print(f"✗ {package} missing")
-    
+
     if missing_packages:
         print(f"\nInstall missing packages: pip install {' '.join(missing_packages)}")
         return False
-    
+
     # Check API keys
     api_keys = {
```

Ruff (0.14.6)

Local variable `api_keys` is assigned to but never used

Remove assignment to unused variable `api_keys`

(F841)